app/templates/components/build-layout.hbs

```diff
@@ -21,7 +21,7 @@
       {{/if}}
     {{else}}
       <JobInfrastructureNotification @job={{this.job}} />
-      <JobTabs @job={{this.job}} @repo={{this.repo}} />
+      <JobTabs @job={{this.job}} @request={{this.job.build.request}} @repo={{this.repo}} />
```

Comment:
Is it necessary to pass `@request` separately? It feels like it could be derived from `this.job` inside of the component

Reply:
moved it back 👍